### PR TITLE
Feature lwdev 6958 withdrawal requested

### PR DESCRIPTION
**Pull request by igromyko**

### CODE REVIEW

File: src/Lykke.Job.Messages.Services/Email/EmailTemplateProvider.cs [hunk 1]
```diff
@@ -42,6 +42,10 @@ string MatchEvaluator(Match match)
                     var key = match.Groups[1].Value;
                     if (parameters != null && parameters.ContainsKey(key))
                         return parameters[key];
+                    if (key == "Year")
```
Comment (paradise) [inline src/Lykke.Job.Messages.Services/Email/EmailTemplateProvider.cs]: there is no need for this

File: src/Lykke.Job.Messages.Contract/Lykke.Job.Messages.Contract/Lykke.Job.Messages.Contract.csproj [hunk 2]
```diff
@@ -4,10 +4,6 @@
     <Version>1.0.1</Version>
   </PropertyGroup>
 
-  <ItemGroup>
```
Comment (paradise) [inline src/Lykke.Job.Messages.Contract/Lykke.Job.Messages.Contract/Lykke.Job.Messages.Contract.csproj]: is it ok ?

Comment (igromyko) [reply src/Lykke.Job.Messages.Contract/Lykke.Job.Messages.Contract/Lykke.Job.Messages.Contract.csproj]: it works without this section.
Seems ok.

File: src/Lykke.Job.Messages.Services/Email/EmailTemplateProvider.cs (hunk 1 above)
Comment (igromyko) [reply src/Lykke.Job.Messages.Services/Email/EmailTemplateProvider.cs]: deleted

**Pull request merged**